scripts/apps/archive/controllers/BaseListController.ts

```diff
@@ -55,12 +62,13 @@ export class BaseListController {
         return query;
     }
 
-    fetchItems(criteria) {
+    fetchItems(criteria, next?) {
         console.warn('No API defined.');
     }
 
     fetchItem(id) {
         console.warn('no api defined');
+        return new Promise((r) => r());
```

Comment:
that's like `return Promise.resolve()`; which is imo more readable

Reply:
@petrjasek yeah agree, I didn't know you could do that actually